src/main/java/com/wcc/platform/domain/platform/Filters.java

```diff
@@ -0,0 +1,25 @@
+package com.wcc.platform.domain.platform;
+
+import com.wcc.platform.domain.cms.attributes.EventDays;
+import com.wcc.platform.domain.cms.attributes.EventType;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+/** Filters model. */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Getter
+public class Filters {
+  private List<EventType> type;
```

Comment:
I think also we can filter by text, WDT?

Reply:
What do you mean by filter by text ? Filters are always predefined

Reply:
yes and no, because I was considering that we can use the same class to include into Event as param to filter events, and for this case it is missing the text, but that could be solved in another way. So just ignore for now.